boot_partition.txt

```diff
@@ -0,0 +1,812 @@
+[   57467608]  ./flatcar/vmlinuz-a*
+[     949440]  ./EFI/boot/bootx64.efi*
+[     855912]  ./EFI/boot/mmx64.efi*
+[     783864]  ./flatcar/grub/x86_64-efi/core.efi.signed*
+[     783864]  ./EFI/boot/grubx64.efi*
+[     782336]  ./flatcar/grub/x86_64-efi/core.efi*
+[     470624]  ./xen/pvboot-x86_64.elf*
+[     470624]  ./flatcar/grub/x86_64-xen/core.elf*
```

Comment:
Same file, and i'm not convinced pvboot elf works. Hasn't been tested in years/ever.